pkg/odo/cli/version/version.go

```diff
@@ -63,7 +64,7 @@ func (o *VersionOptions) Complete(cmdline cmdline.Cmdline, args []string) (err e
 			// checking the value of timeout in preference
 			var timeout time.Duration
 			if o.clientset.PreferenceClient != nil {
-				timeout = time.Duration(o.clientset.PreferenceClient.GetTimeout()) * time.Second
+				timeout = o.clientset.PreferenceClient.GetTimeout() * time.Second
```

Comment:
I don't think you need to multiply by 1s (same in line 70)

Reply:
You're right, I missed this one.